WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentOnboardingViewModel.swift

```diff
@@ -3,7 +3,7 @@ import Yosemite
 import Experiments
 import WooFoundation
 
-final class InPersonPaymentsViewModel: ObservableObject, PaymentSettingsFlowPresentedViewModel {
+final class CardPresentPaymentOnboardingViewModel: ObservableObject, PaymentSettingsFlowPresentedViewModel {
```

Comment:
[See my comment here](https://github.com/woocommerce/woocommerce-ios/issues/12863#issuecomment-2139659075) then pick whether you prefer "s" or no "s"!